### PR TITLE
Enable normquant output register to cut critical path

### PR DESCRIPTION
**Pull request by LuigiGhionda**
This PR adapt the FSM of the accumulator in order to work with the latency introduced by a pipeline register in the normquant module. This register is useful to cut a critical path that may arise during the physical implementation of the design.

This PR is marked as a draft since two tasks still need to be performed:
- [x] Add parametric generation to distinguish the case when the register is instantiated and when it's not.
- [x] Check from P&R if timing issues have been solved.

### CODE REVIEW

File: Bender.yml
Comment (FrancescoConti) [inline Bender.yml:1]: which version is this?

File: rtl/neureka_streamer.sv
Comment (FrancescoConti) [inline rtl/neureka_streamer.sv:1]: Why are these changes necessary and related to this PR?

File: rtl/verif/tb_neureka.sv
Comment (FrancescoConti) [inline rtl/verif/tb_neureka.sv:1]: isn't this more related to the "hidden error" problem?

**Review by FrancescoConti — COMMENTED** [review]
Mostly ok, but a few changes do not seem related to this PR specifically. Are they part of the other PRs?

**Comment by LuigiGhionda** [comment]
I created the `astral-v1.0` branch to have a separate branch where to merge all the modifications intended for Astral (that maybe we don't want to merge into the main branch). Probably, in this case, it's a helpful modification in all cases. Let me know, and I will switch the PR target branch to the main.

**Comment by LuigiGhionda** [comment]
Changed the target branch to `main`
